### ISSUE TITLE
Can you bump refreshing of Castle Game Engine RSS feed

### ISSUE THREAD

**Issue by michaliskambi** [FreeGameDev/planet-config#3]
Hi,

Castle Game Engine feed is currently "down" on http://planetdev.freegamedev.net/ .

But inside the config it is correctly referenced: https://github.com/FreeGameDev/planet-config/blob/master/fgdplanet-dev-feeds.ini shows:

~~~~
[http://castle-engine.sourceforge.net/news_feed.php]
name = Castle Game Engine
~~~~

The http://castle-engine.sourceforge.net/news_feed.php redirects to https://castle-engine.sourceforge.io/wp/feed/ which shows our correct RSS feed, produced by Wordpress now. This is good, it works reliably and we want to use it.

However, a week ago http://castle-engine.sourceforge.net/news_feed.php was redirecting to a different place, http://gplusrss.com/rss/feed/c3dd3cd5ec6725d92b8b5b0633b2daa852f29730ecc6b , that was presenting a Google+ posts as an RSS feed. It was making Internal Errors a week ago (although they are fixed now).

However, http://planetdev.freegamedev.net/ seems stuck thinking that Castle Game Engine feed causes an internal error. The tooltip over it shows https://plus.google.com/101185352355602218697 , so it seems that http://planetdev.freegamedev.net/

- Still remembers the old redirect: http://castle-engine.sourceforge.net/news_feed.php -> http://gplusrss.com/rss/feed/c3dd3cd5ec6725d92b8b5b0633b2daa852f29730ecc6b
- And it remembers that http://gplusrss.com/rss/feed/c3dd3cd5ec6725d92b8b5b0633b2daa852f29730ecc6b fails, even though it seems to work now..

If the problem is that our news feed is served through https (as I see issue https://github.com/FreeGameDev/planet-config/issues/1 ), I can provide a non-encrypted http mirror of it.

**Comment by akien-mga** [comment]
Hi @michaliskambi 

I believe the issue might be due to the https protocol as you mentioned. If you can provide a http feed, I'll gladly update the config (or you can make a PR).

**Comment by michaliskambi** [comment]
The http feed of Castle Game Engine is now available on

  http://michalis.ii.uni.wroc.pl/castle-engine-feed

Can you update the config? Thank you!

**Comment by akien-mga** [comment]
Fixed with c4d63ff (the actual planet config still needs to be synced with this repo though, but I'll ask the admin to do it).
